packages/pn-commons/src/components/SnackBar/SnackBar.tsx

```diff
@@ -0,0 +1,86 @@
+import { Alert, Snackbar, IconButton } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import { useEffect, useState } from 'react';
+import { useIsMobile } from '../../hooks/useIsMobile';
+import { MessageType } from '../../types/MessageType';
+
+type Props = {
+  /** whether the sneakbar should be open or not */
+  open: boolean;
+  /** message type (error, success, info, warning) */
+  type: MessageType;
+  /** message to be shown */
+  message: React.ReactNode;
+  /** A closing delay: if specified the sneakbar would close itself */
+  closingDelay?: number;
+  /** onClose action */
+  onClose?: () => void;
+  variant?: 'outlined' | 'standard' | 'filled';
+};
+
+const SnackBar = ({ message, open, type, closingDelay, onClose, variant = 'outlined' }: Props) => {
+  const [openStatus, setOpenStatus] = useState(open);
+  const isMobile = useIsMobile();
+
+  const closeSnackBar = () => {
+    setOpenStatus(false);
+    if (onClose) {
+      onClose();
+    }
+  };
+
+  useEffect(() => {
+    if (closingDelay && openStatus) {
+      const timer = setTimeout(() => {
+        closeSnackBar();
+      }, closingDelay);
+      return () => clearTimeout(timer);
+    }
+    return;
```

Comment:
Is this needed? If so, please add a comment to the code explaining why.

Reply:
do it